src/fusion.cpp

```diff
@@ -430,6 +474,7 @@ miopenStatus_t FusionPlanDescriptor::Compile(Handle& handle)
     auto fusion_ctx       = FusionContext{handle};
     auto fusion_problem   = FusionDescription{this};
     fusion_ctx.DetectRocm();
+    // tmp_sols is collection of all the solvers that isApplicable for the fusion_problem.
     const auto tmp_sols = solvers.SearchForAllSolutions(
         fusion_ctx, fusion_problem, miopen::GetDb(fusion_ctx), AnyInvokeParams{});
     std::vector<miopen::solver::ConvSolution> sols;
```

Comment:
When you say "_You should filter them by adding direction to your context and checking it in IsApplicable, rather than manually filtering find results_ " similar to [this](https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/src/solver/conv_asm_1x1u_bias_activ_fused.cpp#L251)?

Reply:
Probably. I am not familiar with fusion types, but any post-find filtering would implicitly consume a lot of time due to redundant solvers find being executed at the first place. But this is out of the scope of PR as well, I guess. Just wanted to left a note here, because this can be fixed later and that would improve host performance on average.